### PR TITLE
WIP: Qt5 deprecated functions @jcfr 

### PR DESCRIPTION
**Pull request by dzenanz**
@jcfr

### CODE REVIEW

**Comment by lassoan** [comment]
@jcfr When the 4.8.1 patch is planned to be released?

File: Utilities/Scripts/SlicerWizard/__version__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 __version_info__ = (
-  4,
-  9,
+  1,
+  1,
```
Comment (cpinter) [inline Utilities/Scripts/SlicerWizard/__version__.py]: Was this change intentional? It does not seem to be necessary.

File: Modules/Scripted/DICOMLib/DICOMWidgets.py [hunk 2]
```diff
@@ -516,8 +516,12 @@ def promptForDatabaseDirectory(self):
       try:
         os.makedirs(databaseDirectory)
       except OSError:
-        documentsLocation = qt.QDesktopServices.DocumentsLocation
-        documents = qt.QDesktopServices.storageLocation(documentsLocation)
+        # Qt4
+        # documentsLocation = qt.QDesktopServices.DocumentsLocation
+        # documents = qt.QDesktopServices.storageLocation(documentsLocation)
+        # Qt5
```
Comment (pieper) [inline Modules/Scripted/DICOMLib/DICOMWidgets.py]: I'd prefer not to have the old code in comments - I don't see that it serves any purpose going forward.

File: Utilities/Scripts/SlicerWizard/__version__.py (hunk 1 above)
Comment (dzenanz) [reply Utilities/Scripts/SlicerWizard/__version__.py]: It was probably not. But JC was debugging stuff on my computer while I was having lunch :smile: so I thought I would commit all the changes. @jcfr said he will do additional testing and clean up the fix.

File: Modules/Scripted/DICOMLib/DICOMWidgets.py (hunk 2 above)
Comment (dzenanz) [reply Modules/Scripted/DICOMLib/DICOMWidgets.py]: Maybe I should have marked this as `WIP:`?

Comment (lassoan) [reply Modules/Scripted/DICOMLib/DICOMWidgets.py]: Yes, next time add a comment: "Please don't even review this yet" :)
(or just push it to your own fork?)

**Comment by jcfr** [comment]
Apologize for the noise.

**Comment by jcfr** [comment]
Closing. Superseded by https://github.com/Slicer/Slicer/pull/853

**Comment by jcfr** [comment]
> When the 4.8.1 patch is planned to be released?

I will post an update on discourse: https://discourse.slicer.org/t/slicer-4-8-1-release-and-transition-to-qt5-and-vtk9/1551
